### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by dword4**
Cleaning up things preventing Molecule 3.x from working with this

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Role Name
```
Comment (sblack4) [inline README.md]: Are you going to fill this out?

File: meta/main.yml [hunk 2]
```diff
@@ -1,27 +1,57 @@
...
-      versions:
-        - all
-  categories:
-    - monitoring
-  galaxy_tags:
-    - ossec
-    - ossec-agent
-    - hids
-    - monitoring
+  author: your name
```
Comment (sblack4) [inline meta/main.yml]: This too, is default molecule stuff

**Review by sblack4 — CHANGES_REQUESTED** [review]
I may not understand the scope of this PR but it looks like there's some metadata that is still default. IDK if the idea is to make that a future PR.

File: README.md (hunk 1 above)
Comment (dword4) [reply README.md]: Should be updated in c64e407b40169f724ea038614fe77dad27c4b662 to have the useful information

File: meta/main.yml (hunk 2 above)
Comment (dword4) [reply meta/main.yml]: Also addressed in c64e407b40169f724ea038614fe77dad27c4b662 to have correct information

File: vars/main.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+# vars file for my-new-role
+ossec_server_ip: 192.168.1.11
+ossec_server_name: ossec-master.ops.carejourney.com
```
Comment (sblack4) [inline vars/main.yml]: I don't think we want to commit client data to github

**Review by sblack4 — APPROVED** [review]
close - just one last thing

**Pull request merged**